src/main/kotlin/no/nb/mlt/wls/product/payloads/SynqProductPayload.kt

```diff
@@ -26,7 +26,7 @@ data class SynqProductPayload(
         ESK
     }
 
-    data class Barcode(val barcodeId: String)
+    data class Barcode(val barcodeId: String, val barcodeDesc: String)
```

Comment:
Think it's actually better to delete barcodeDescription in the dummy-synq. I forgot to do it with all the meetings going on. Since it's not required and we don't seem to have a use for it.

Reply:
Will make a PR for this :+1: